docs/faq.md

```diff
@@ -0,0 +1,68 @@
+# F.A.Q.
+
+Here are a couple of common questions about JupyterLab and the transition from
+the classic Notebook interface.
+
+## Is the classic notebook going away ?
+
+No, at least not yet, it will though get way less updates. There will be a
```

Comment:
No, not in the short term. The classic notebook will progressively receive less updates over time.

Reply:
I would put this part also in the README